### PR TITLE
Replaced Binance API with Kraken example

### LINKED ISSUE #1375: Binance API does not work when downloading in publish-flow-restapi.md
After publishing the rest API asset on mumbai, file info couldn't be fetched from provider and it crashes at the download part (in Ocean Market and also from ocean.py console) https://market.oceanprotocol.com/asset/did:op:c5cc054e0e1eab4a0e4a6669e697c14b27061d0aca6fd4c01f2a767a331b9376. It is due to the URl, because Binance API has some limitations and it's not available anymore in some places.

Created another one based on Kraken API URL and it worked https://market.oceanprotocol.com/asset/did:op:3447dc6a8761ec5b22d55510ff39d734e9f89b61eee1772628e08dc21a0342b5

Update the URL from the README to avoid the errors.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #1375  .

Changes proposed in this PR:

- replaced Binance API with Kraken example.

### CODE REVIEW

**Review by trentmc — CHANGES_REQUESTED** [review]

File: READMEs/publish-flow-restapi.md [hunk 1]
```diff
@@ -64,7 +61,7 @@ file_name = os.path.join(asset_dir, 'file0')
 
 Now, load the file and use its data.
 
-The data follows the Binance docs specs for Kline/Candlestick Data, [here](https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-data).
+The data follows the Kraken docs specs for Data, [here](https://docs.kraken.com/rest/#section/Example-API-Clients).
```
[the remark this answers is not in the document]
Comment (mariacarmina) [reply READMEs/publish-flow-restapi.md:64]: Thank you, will do.

[the remark this answers is not in the document]
Comment (trentmc) [reply READMEs/publish-flow-restapi.md:64]: It's not done yet. I don't want just "data". I want the closing prices, just like the past example.

[the remark this answers is not in the document]
Comment (mariacarmina) [reply READMEs/publish-flow-restapi.md]: Ok, I have also used [ticker information](https://docs.kraken.com/rest/#tag/Market-Data/operation/getTickerInformation) for this just as it was with Binance API.

**Review by calina-c — APPROVED** [review]

**Review by trentmc — CHANGES_REQUESTED** [review]
I do *not* approve yet.

I want it to grab not just the most recent close price, but the last 500 (or whatever it is). Like Binance did.

Please fix.

**Comment by mariacarmina** [comment]
The closing prices output after running the README steps:
Screenshot from 2023-04-05 17-26-58

File: READMEs/publish-flow-restapi.md [hunk 2]
```diff
@@ -64,24 +67,24 @@ file_name = os.path.join(asset_dir, 'file0')
...
+    data_str = file.read().rstrip().replace("'", '"')
+
+import json
+data = json.loads(data_str)
+
+# Data is a list of lists
+# -Outer dictionary contains 2 keys, one for errors and one for the result with the pair.
+# -Inner dictionary have 9 entries each: Kline open time, Open price, High price, Low price, close Price, Vol, ..
+# Get close price
+close_prices = [float(item[4]) for item in data['result'][pair]]
```
Comment (trentmc) [inline READMEs/publish-flow-restapi.md:87]: Replace `item` with `data_at_day`

**Review by trentmc — CHANGES_REQUESTED** [review]
Getting close, good! 

One small tweak requested.

**Review by trentmc — APPROVED** [review]

**Pull request merged**